commands.go

```diff
@@ -207,6 +211,27 @@ func NewCommandDeriveKey(pathStr string) (*apdu.Command, error) {
 	), nil
 }
 
+func NewCommandLoadKey(isSeed bool, isExtended bool, payload []byte) (*apdu.Command) {
+	var p1 uint8
+	var data []byte
+	if isSeed == true {
+		p1 = P1LoadKeySeed
+		data = payload
```

Comment:
maybe I missed it before, but I see we just send data/payload. but as explained in the docs, if P1 is 1 or 2 the data sent must be encoded in tlv using the 0xA1 tag for the template.

https://status.im/keycard_api/apdu_loadkey.html

Reply:
so maybe we can have NewCommandLoadKey(type LoadType, data []byte) where LoadType are constants defining keypair/ext-keypair/seed. and based on that we define the right p1 and data format. 
What do you think @alex-miller-0 ?
I can also continue with this if you don't have a lot of time.

Reply:
My idea was to punt data encoding to another layer, since I didn't see an example of TLV encoding in this codebase. If you want that logic here, I probably shouldn't be doing it since I'm not sure what types of data structures or libraries you want to use.

Feel free to close this PR and use any of the code in another 👍